### PR TITLE
Feature/successful match event

### PR DESCRIPTION
**Pull request by Jiacomina**
Add collision detection event to falling blocks

### CODE REVIEW

File: assets/Script/game/fallingBlock.js [hunk 1]
```diff
@@ -109,13 +109,21 @@ cc.Class({
...
   update(dt) {
-    if (this.movingVertical) {
-      this.node.y -= this.speed * dt;
+    this.node.y -= this.speed * dt;
+  },
+
+  onCollisionEnter(other, self) {
+    if (other.node.name === 'catcher1') {
+      // other.node.zIndex = 1;                          // +1 catcher zindex - block goes behind catcher
+      // const fadeOut = cc.fadeOut(this.fadeOutTime);
```
Comment (Jesse-Cameron) [inline assets/Script/game/fallingBlock.js]: would you mind removing these old comments?

File: assets/Script/game/fallingBlock.js [hunk 2]
```diff
@@ -109,13 +109,21 @@ cc.Class({
   },
 
   update(dt) {
-    if (this.movingVertical) {
```
Comment (Jesse-Cameron) [inline assets/Script/game/fallingBlock.js]: This line will need to be re-added.
Essentially, when the block is being dragged the falling animation will stop.

File: assets/Script/game/fallingBlock.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ cc.Class({
   extends: cc.Component,
 
   properties: {
-    speed: 50,
-    movingHorizontal: true,
-    movingVertical: true
+    speed: 200,
+    fadeOutTime: 0.4,
+    blockType: 'blue'
```
Comment (Jesse-Cameron) [inline assets/Script/game/fallingBlock.js]: Until we settle on how to do the 'block matching', might be worth leaving this out.

**Review by Jesse-Cameron — CHANGES_REQUESTED** [review]
Overall, works perfectly. Just a couple of minor semantic changes!

**Comment by Jiacomina** [comment]
Thanks Jesse, changes have been made

**Review by Jesse-Cameron — APPROVED** [review]
:fireworks:

**Pull request merged**